docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -191,7 +191,13 @@ module.exports = {
     artifactsDir: 'build/contracts', // Directory where contract artifacts are stored
   },
 
-  blockGasLimit: 8e6, // Maximum gas per block
+  node: { // Options passed directly to Ganache client
+    gasLimit: 8e6, // Maximum gas per block
+    gasPrice: 20e9 // Sets the default gas price for transactions if not otherwise specified.
+    fork: 'https://mainnet.infura.io/v3/{token}@{blocknumber}, // An url to Ethereum node to use as a source for a fork
+    unlocked_accounts: ['0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B'], // Array of addresses specifying which accounts should be unlocked.
```

Comment:
I'd document these separately: `unlocked_ accounts` makes no sense without `fork`, and I'm not sure how the other interact (can you fork and change `gasLimit`?). 

In any case, `fork` is quite specialized and will only be used by advanced users, I'd rather not cause confusion by listing it along with everything else. See the [Advanced Options](https://docs.openzeppelin.com/test-environment/0.1/getting-started#advanced-options) section.

Reply:
Totally agree on Advanced Options.